packages/tss/src/tss.ts

```diff
@@ -461,17 +466,21 @@ export class TKeyTSS extends ThresholdKey {
 
     // reconstruct final key using sss
     const ec = this._tssCurve;
-    const finalKey = lagrangeInterpolation(ec, [tempShare, factorShare], [new BN(tempIndex), new BN(factorIndex)]);
+    const tssKey = lagrangeInterpolation(ec, [tempShare, factorShare], [new BN(tempIndex), new BN(factorIndex)]);
 
-    // deleted created tss share
+    // delete created tss share
     await this.deleteFactorPub({
       factorKey,
       deleteFactorPub: tempFactorPub,
       authSignatures,
       selectedServers,
     });
 
-    return finalKey;
+    // Derive key for account index.
+    const nonce = this.computeAccountNonce(accountIndex);
+    const derivedKey = tssKey.add(nonce).umod(this._tssCurve.n);
```

Comment:
wont this change the key for account Index 0 ?

Reply:
No, because account index 0 returns `nonce = 0`.

https://github.com/tkey/tkey/blob/92694ca0d5c644e919b3c5089ee5a462ea1d0808/packages/tss/src/tss.ts#L603-L606